index.js

```diff
@@ -21,6 +23,11 @@ if (process.env.USE_BUNYAN_PRETTY_STREAM === 'true') {
   const prettyStdOut = new PrettyStream();
   prettyStdOut.pipe(process.stdout);
   config.streams.push({ type: 'raw', level: loggerLevel, stream: prettyStdOut });
+} else if (process.env.LOGGER_USE_SENSITIVE_DATA_STREAM !== 'false') {
```

Comment:
Does this mean that if `process.env.USE_BUNYAN_PRETTY_STREAM` is set to `true` then sensitive data won't be filtered at all ?

Reply:
Yes